packages/forma-36-react-components/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -1,53 +1,67 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, boolean } from '@storybook/addon-knobs';
-import { action } from '@storybook/addon-actions';
 
-import Checkbox from './Checkbox';
+import Checkbox, { CheckboxProps } from './Checkbox';
 import Flex from '../Flex/Flex';
 import SectionHeading from '../Typography/SectionHeading';
 
-storiesOf('Components/Checkbox', module)
-  .addParameters({
-    propTypes: Checkbox['__docgenInfo'],
-    component: Checkbox,
-  })
-  .add('default', () => (
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  parameters: {
+    propTypes: [Checkbox['__docgenInfo']],
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+    checked: { control: { disable: false } },
+    required: { control: { disable: false } },
+    disabled: { control: { disable: false } },
+  },
+};
+
+export const basic = (args: CheckboxProps) => (
+  <Checkbox id="Checkbox" labelText={args['labelText']} name={args.name} />
```

Comment:
I think a better way of doing this is by spreading the args, like:

```jsx
<Checkbox {...args} />
```

otherwise, every time a prop is added or removed you will need to refactor this